### DISCUSSION TITLE
Student developer pack application gets rejected

### DISCUSSION THREAD

**Discussion by 85-Astatine** [community/community#125069] [GitHub Education]
### Select Topic Area

Question

### Body

i want to activate student developer pack, currently im at my home(away from campus), what should i do?(github detects that im not near campus an rejects my application)

**Comment by nikhilsbuilds** [comment]
To activate the GitHub Student Developer Pack when you're away from your campus, you can try the following steps:

1. **Provide Proof of Enrollment**: Since GitHub can't verify your student status automatically when you're off-campus, you'll need to provide proof of enrollment. You can usually do this by uploading a document like a student ID card, enrollment letter, or transcript.

2. **Use Your School Email Address**: Make sure you're using your school-issued email address when applying for the Student Developer Pack. GitHub often requires a valid .edu email address to verify your student status.

3. **Update Your Location**: In your GitHub account settings, you may be able to update your location to your school's address or campus location. This could help GitHub recognize that you're a student at that institution.

4. **Contact GitHub Support**: If you've tried the above steps and your application is still being rejected, you may need to contact GitHub Support directly. Explain your situation and provide any necessary documentation to prove your student status. They may be able to manually review and approve your application.

Keep in mind that GitHub's verification process can vary, and they may have additional requirements or steps depending on your specific situation. Being patient and providing clear documentation of your enrollment status is often the best approach when applying remotely.
